io/file_test.go

```diff
@@ -425,7 +447,9 @@ func (s *TestSuite) TestGetEmptyFiles(c *C) {
 
 func (s *TestSuite) TestReadFileContent(c *C) {
 	fakeDir := fake.CreateTempDirectory("", c)
-	defer os.RemoveAll(fakeDir)
+	defer func() {
+		_ = os.RemoveAll(fakeDir)
+	}()
```

Comment:
Why use `defer func() {...}()` instead?

Reply:
This is for code check only to explicitly ignore the returned variable. There is no extra effect.